Task2/controllers/users.js

```diff
@@ -0,0 +1,89 @@
+const usersData = require('./../data/users.json');
+import { ErrorMessages } from './../config/constants';
+import { v4 as uuidv4 } from 'uuid';
+import { omit } from 'lodash';
+
+const getUserById = (req, res) => {
+  const id = req.params.userId;
+  let user = usersData.data.find((item) => item.id === id);
+  if (!user) {
+    res.send({
+      error: ErrorMessages.NOUSER,
+    });
+  }
+  res.send(user);
+};
+
+const removeUserById = (req, res) => {
+  const id = req.params.userId;
+  let userIndex = usersData.data.findIndex((item) => item.id === id);
+  if (userIndex === -1) {
+    res.send({
+      error: ErrorMessages.NOUSER,
+    });
+  }
+  usersData.data[userIndex].isDeleted = true;
+  res.send({
```

Comment:
response format should follow similar standard for all APIs

Reply:
Not applicable